robottelo/cli/gpgkey.py

```diff
@@ -38,25 +38,23 @@ def info(cls, options=None):
         cls.command_sub = 'info'
 
         result = cls.execute(
-            cls._construct_command(options), output_format='csv')
+            cls._construct_command(options), output_format='json')
+
+        if isinstance(result, dict) or len(result) == 0:
```

Comment:
Not always. Some times you get a list. My first try was exactly like yours, but it make some tests failing